control-plane-operator/controllers/hostedcontrolplane/kas/deployment.go

```diff
@@ -429,10 +429,17 @@ func buildKASContainerMain(image string, port int32, noProxyCIDRs []string, hcp
 		c.Command = []string{
 			"hyperkube",
 		}
+
+		// default to level of 2
+		kasVerbosityLevel := "-v2"
+		if hcp.Annotations[hyperv1.KubeAPIServerVerbosityLevelAnnotation] != "" {
```

Comment:
We should parse this as an integer and ignore it or error if it's not one - passing `--v=whatever` will crash the KAS.